data/crac-creation/crac-creator-api/src/main/java/com/farao_community/farao/data/crac_creation/creator/api/parameters/CracCreationParameters.java

```diff
@@ -15,7 +15,8 @@
 import com.powsybl.commons.extensions.ExtensionConfigLoader;
```

Comment:
GENERAL : could you please update the PR description ? it sometimes comes in handy looking back on closed PRs